.swiftlint.yml

```diff
@@ -48,7 +48,7 @@ opt_in_rules:
   - empty_string
   - identical_operands
   - inert_defer
-  - indentation_width
+#  - indentation_width #TODO in BM-694
```

Comment:
I don't have a strong preference here, but having two comment markers in one line seems weird.
```suggestion
# TODO in BM-694:   indentation_width
```

Reply:
Thanks but I'll leave it as it is :)